content/guides/10-minute-tutorial.md

```diff
@@ -293,6 +293,13 @@ class RunCucumberTest
 {{% block "javascript" %}}
 We'll start by creating a new directory and an empty Node.js project.
 
+Sometime, before that we need to install cucumber globally by running this command:
```

Comment:
As this is Linux specific, could you add it inside a note block with a title like "If you are using Linux"

Reply:
Yeah sure...how about like this ?
![selection_020](https://user-images.githubusercontent.com/39081741/52275241-fed7a000-2974-11e9-853a-b900c534a4bc.png)

Reply:
Sorry, maybe I was not specific enough. I meant similar to the "Example mapping" block:
```
{{% tip "Example Mapping"%}}
Try running an [Example Mapping](/bdd/example-mapping) workshop in your team to
design examples together.
{{% /tip %}}
```

Reply:
But with the information you added, ofcourse.